code/unit_tests/~unit_test_types.dm

```diff
@@ -66,7 +71,9 @@
 	name = "Edge of Map - Only map space turfs here"
 
 /obj/item/paper/secret_note/example
+	is_spawnable_type = FALSE
```

Comment:
Consistency, and also they *were* runtiming because I was too lazy to copy over the example secrets to my local testing. I did that though, so it's technically not an issue. That said, I don't want the create-and-destroy tests to fail if the secrets system breaks, and this is already tested earlier.